### PR TITLE
added some waifus from steins:gate, plus a jojo

### PR DESCRIPTION
**Pull request by lucasdpau**
First time doing a 'real' PR, let me know if I should do anything else.

### CODE REVIEW

**Comment by yourWaifu** [comment]
Looks good thank you.

File: waifus/kurisu makise.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿{
+	"type" : 0,
+	"japanese-name" : "牧瀬 紅莉栖",
+	"english-name" : "Kurisu Makise",
+	"year" : 1992,
+	"month" : 7,
+	"day-of-month" : 25,
+	"age-in-show": 18,
+	"image" : "https://steins-gate.fandom.com/wiki/Kurisu_Makise?file=Kurisu%20profile.png",
```
Comment (yourWaifu) [inline waifus/kurisu makise.json]: link doesn't go to the image 
image
use this one https://vignette.wikia.nocookie.net/steins-gate/images/4/4a/Kurisu_Full_profile.png

File: waifus/mayuri shiina.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿{
+	"type" : 0,
+	"japanese-name" : "椎名 まゆり",
+	"english-name" : "Mayuri Shiina",
+	"year" : 1994,
+	"month" : 2,
+	"day-of-month" : 1,
+	"age-in-show": 16,
+	"image" : "https://vignette.wikia.nocookie.net/steins-gate/images/0/0e/Mayuri_Shiina.png/revision/latest?cb=20181024172638",
```
Comment (yourWaifu) [inline waifus/mayuri shiina.json]: I think this one is better https://vignette.wikia.nocookie.net/steins-gate/images/a/ac/Mayuri_full_profile.png

File: waifus/maho hiyajo.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿{
+	"type" : 0,
+	"japanese-name" : "比屋定真帆",
+	"english-name" : "Maho Hiyajo",
+	"year" : 1989,
+	"month" : 11,
+	"day-of-month" : 2,
+	"age-in-show": 21,
+	"image" : "https://vignette.wikia.nocookie.net/steins-gate/images/3/31/Maho_Hiyajo_profile.png/revision/latest?cb=20151222154143",
```
Comment (yourWaifu) [inline waifus/maho hiyajo.json]: ditto https://vignette.wikia.nocookie.net/steins-gate/images/1/1c/Hiyajomaho.png

File: waifus/moaeka kiryu.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+﻿{
+	"type" : 0,
+	"english-name" : "Moeka Kiryu",
+	"aliases" : ["Shining Finger"],
+	"image" : "https://vignette.wikia.nocookie.net/steins-gate/images/d/de/Moeka_Kiryuu.png/revision/latest?cb=20180718213008",
```
Comment (yourWaifu) [inline waifus/moaeka kiryu.json]: ditto https://vignette.wikia.nocookie.net/steins-gate/images/3/3d/Moeka_Full_Profile-0.png

**Comment by yourWaifu** [comment]
thanks for the pull request, I reviewed your changes and there are a few changes that I like you to make before I merge your changes.

**Comment by lucasdpau** [comment]
Alright I made the changes you requested, I can't believe how sloppy I was with the image links haha

**Comment by yourWaifu** [comment]
thank you

**Pull request merged**